### PR TITLE
Closes #6 Adds project outline

### LINKED ISSUE #6: We need an initial project outline
This will help make more concrete the following:
- what are we going to be building
- what are the components
- what does the architecture look like
- what are our doubts and fears

### PR DESCRIPTION
**Pull request by patchandpray**
I tried to give a general outline of some sections that could be relevant and or helpful in making the project more concrete.

We can use and fill in these sections as we see fit. It should help us give a bit more direction as wel.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,104 @@
...
+
+## A Cloud Native Data Workflow on Kubernetes using Argo Workflows and K8ssandra
+
+This project is about:
+Using cloud-native data services on Kubernetes
+Using a cloud-native database on Kubernetes for storing data
+Using a cloud-native workflow engine to determine how to handle and move data
+Showing how a simple app or script can be used on Kubernetes as part of a data pipeline
+Showing how to develop custom code components for use in Kubernetes
+Showing which depending services you might encounter and need when wanting to start running data workloads on kubernetes
```
Comment (verma-kunal) [inline README.md:14]:
```suggestion
- Using cloud-native data services on Kubernetes
- Using a cloud-native database on Kubernetes for storing data
- Using a cloud-native workflow engine to determine how to handle and move data
- Showing how a simple app or script can be used on Kubernetes as part of a data pipeline
- Showing how to develop custom code components for use in Kubernetes
- Showing which depending services you might encounter and need when wanting to start running data workloads on Kubernetes
```

Comment (verma-kunal) [reply README.md:14]: adding bullet-points would be more precise here, to differentiate each point!!

Comment (patchandpray) [reply README.md:14]: Agreed, makes it more clear!

**Comment by patchandpray** [comment]
I was thinking of also adding some contact information on this page, like when we have a bi-weekly progress session. And how to reach out to us or follow the conversation on the slack channel @verma-kunal what do you think?

**Comment by verma-kunal** [comment]
> I was thinking of also adding some contact information on this page, like when we have a bi-weekly progress session. And how to reach out to us or follow the conversation on the slack channel @verma-kunal what do you think?

really nice idea!! Sounds good 🙂

**Comment by kaiwalyakoparkar** [comment]
> I was thinking of also adding some contact information on this page, like when we have a bi-weekly progress session. And how to reach out to us or follow the conversation on the slack channel @verma-kunal what do you think?

Nice thought! We also have a google form which we provide for people who want to attend the meeting. (Can be found in the pinned messages in the channel)

**Comment by verma-kunal** [comment]
> > I was thinking of also adding some contact information on this page, like when we have a bi-weekly progress session. And how to reach out to us or follow the conversation on the slack channel @verma-kunal what do you think?
> 
> Nice thought! We also have a google form which we provide for people who want to attend the meeting. (Can be found in the pinned messages in the channel)

i think we can direct the person (who would be interested to know more) to our slack & then provide them with the google form link!! 
What do you think?

**Comment by kaiwalyakoparkar** [comment]
> > > I was thinking of also adding some contact information on this page, like when we have a bi-weekly progress session. And how to reach out to us or follow the conversation on the slack channel @verma-kunal what do you think?
> > 
> > 
> > Nice thought! We also have a google form which we provide for people who want to attend the meeting. (Can be found in the pinned messages in the channel)
> 
> i think we can direct the person (who would be interested to know more) to our slack & then provide them with the google form link!!
> What do you think?

Yeah, makes sense.

**Comment by patchandpray** [comment]
> > > > I was thinking of also adding some contact information on this page, like when we have a bi-weekly progress session. And how to reach out to us or follow the conversation on the slack channel @verma-kunal what do you think?
> > > 
> > > 
> > > Nice thought! We also have a google form which we provide for people who want to attend the meeting. (Can be found in the pinned messages in the channel)
> > 
> > 
> > i think we can direct the person (who would be interested to know more) to our slack & then provide them with the google form link!!
> > What do you think?
> 
> Yeah, makes sense.

Agreed. I will add the contact information

**Comment by patchandpray** [comment]
@verma-kunal @kaiwalyakoparkar I added the contact info, can you finalise review?

**Review by verma-kunal — COMMENTED** [review]
LGTM 👍

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket:

**Pull request merged**